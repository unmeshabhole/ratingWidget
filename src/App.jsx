import "./App.scss";
import Rating from "./components/Rating.jsx";

function App() {
  return (
    <div className="App">
      <Rating />
      <Rating />
    </div>
  );
}

export default App;
