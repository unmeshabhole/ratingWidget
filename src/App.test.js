import React from "react";
import { shallow } from "enzyme";
import App from "./App";

test("App component html matches the snapshot", () => {
  const component = shallow(<App />);

  expect(component.getElements()).toMatchSnapshot();
});

test("App component should render a div with classname App", () => {
  const wrapper = shallow(<App />);
  expect(wrapper.find(".App")).toHaveLength(1);
});
