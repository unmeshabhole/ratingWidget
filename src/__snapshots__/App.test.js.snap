// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component html matches the snapshot 1`] = `
Array [
  <div
    className="App"
  >
    <Rating />
    <Rating />
  </div>,
]
`;
